Frameworks/UIKit/UITextField.mm

```diff
@@ -100,6 +100,10 @@ - (void)setText:(NSString*)text {
             _passwordBox.password = _text;
         } else {
             _textBox.text = _text;
+            // Ensure caret at end of field in case we programmatically
+            // gain focus (becomeFirstResponder) after the text is set:
+            _textBox.selectionStart = [_text length];
+            _textBox.selectionLength = 0;
```

Comment:
thanks for validating

---
In reply to: [63553197](https://github.com/Microsoft/WinObjC/pull/502#discussion_r63553197) [](ancestors = 63553197)